### ISSUE TITLE
Hi,

### ISSUE THREAD

**Issue by hhhzzj** [dongyp13/Non-Targeted-Adversarial-Attacks#11]
Hi,
I am so interested in your work, and i have a problem when i test the Non-target attack experiment. 
"We choose the images containing just one for each category from ImageNet validation set. They can also be correctly classified by the models. You can use the dataset from [https://github.com/tensorflow/cleverhans/tree/master/examples/nips17_adversarial_competition/dataset](https://github.com/tensorflow/cleverhans/tree/master/examples/nips17_adversarial_competition/dataset) for evaluation."
_Originally posted by @dongyp13 in https://github.com/dongyp13/Non-Targeted-Adversarial-Attacks/issues/6#issuecomment-446959569_
But i find the dataset doesn't contain 1000 images from each category, there are some repeat images in this dataset.

**Comment by dongyp13** [comment]
Hi, thanks for your interest. Actually we use a subset of ImageNet validation set containing 1000 images, which is different from the NIPS17 competition dataset. Our dataset can be downloaded at http://ml.cs.tsinghua.edu.cn/~yinpeng/adversarial/dataset.zip. You can alternatively use the NIPS17 dataset.

**Comment by dongyp13** [comment]
Hi, thanks for your interest. Actually we use a subset of ImageNet validation set containing 1000 images, which is different from the NIPS17 competition dataset. Our dataset can be downloaded at http://ml.cs.tsinghua.edu.cn/~yinpeng/adversarial/dataset.zip. You can alternatively use the NIPS17 dataset.

**Comment by dongyp13** [comment]
I've updated the README regarding this issue.

**Comment by hhhzzj** [comment]
Wow, thank you so much for dealing with my problem!!!
